layouts/basic/modules/Vtiger/uitypes/MultiImage.tpl

```diff
@@ -10,12 +10,12 @@
 		   {if !empty($SPECIAL_VALIDATOR)}data-validator={\App\Json::encode($SPECIAL_VALIDATOR)}{/if}>
 	<div class="border rounded p-2 clearfix c-multi-image">
 		<button class="fileinput-button btn btn-sm btn-primary mr-1 mb-1 float-left c-multi-image__file-btn">
-			<input class="c-multi-image__file" type="file" name="files[]"
+			<input class="c-multi-image__file" type="file" name="{$FIELD_MODEL->getFieldName()}_temp[]"
 				   data-url="file.php?module={$FIELD_MODEL->getModuleName()}&action=MultiImage&field={$FIELD_MODEL->getFieldName()}"
 				   multiple>
-			<input type="hidden" name="hashes[]">
 			<i class="fa fa-plus"></i> {\App\Language::translate('BTN_ADD_FILE', $MODULE_NAME)}
 		</button>
+		<input type="hidden" class="c-multi-image__values" name="{$FIELD_MODEL->getFieldName()}[]" value="[]">
```

Comment:
here is json with files like `[{id:'3', name:'filename.jpg', size:'230 KB' }, ...]`